src/tools/loadData.js

```diff
@@ -0,0 +1,41 @@
+const data = require('../../data/yolo3.json');
+
+let read = () => {
```

Comment:
`const` instead of `let` ya que la variable `read` no va a cambiar de valor

Reply:
Hecho